### DISCUSSION TITLE
I'm definitely doing something wring, hoping someone can help!

### DISCUSSION THREAD

**Discussion by SebSaga** [Extraltodeus/advanced-loopback-for-sd-webui#8] [Q&A] [answered]
First of all, this script is incredible and I appreciate all the hard work people put in t make things like this for the community.

OK so, gratitude out of the way, I'm running into an issue where my images get progressively darker and I'm not sure what setting to change to fix that. I thought adding saturation would help but it didn't. Anyone have any idea why this might be happening?

**Comment by Extraltodeus** [comment, marked as answer]
Load a VAE :
https://huggingface.co/stabilityai/sd-vae-ft-mse-original/tree/main

Also oscillating in between high/low denoising strength fixes this with this extension.

**Reply by Extraltodeus** [reply]
Note :
There is also the color correction option in the regular settings, there is also the custom color correction on that script related to that very issue (disable the inbuilt one before using that one). You can also use the desaturation slider (without any color correction option enabled) to mitigate that effect.

In my experience this happens when the prompt is too different from the source image. So either use an image that has been generated by the same model that you're using, or raise your denoising strength.

**Reply by SebSaga** [reply]
thank you loading in a VAE fixed the issue :) im assuming the option "Use sine denoising strength variation" is for oscillating. Do i just check that box and the denoising then just oscillates? I appreciate you taking the time to answer my dumb questions haha!
